### PR TITLE
(un)Borked jaeger and PAS armor values

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
Changes jaeger armor and PAS values to the intended after the armor variety update.

## Why It's Good For The Game
Not intended to be way it was, changing the borked values to something that was intended.

## Changelog
:cl:
balance: balances PAS armor to be more like base medium jaeger.
balance: all jaeger arm and legpieces are now the same with only cosmetic differences.
balance: reduces jaeger armor to intended, they got a small speed buff in return.
/:cl:

### CODE REVIEW

File: code/modules/modular_armor/armor_pieces.dm [hunk 1]
```diff
@@ -171,22 +168,18 @@
...
 
 /obj/item/armor_module/armor/arms/marine/assault
-	name = "\improper Jaeger Pattern Heavy Assault arm plates"
-	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides high protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Assault armor piece."
-	icon_state = "assault_arms"
-	soft_armor = list("melee" = 10, "bullet" = 15, "laser" = 15, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 10)
-	slowdown = 0.2
+	name = "\improper Jaeger Pattern  Assault arm plates"
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Assault armor piece."
+	icon_state = "assault_arms"
```
Comment (Hughgent) [inline code/modules/modular_armor/armor_pieces.dm:185]: Needs a new line at the end.

File: code/modules/modular_armor/armor_pieces.dm [hunk 2]
```diff
@@ -129,25 +130,21 @@
...
 
 /obj/item/armor_module/armor/legs/marine
-	name = "\improper Jaeger Pattern Medium Infantry leg plates"
-	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides moderate protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece."
+	name = "\improper Jaeger Pattern Infantry leg plates"
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece."
 	icon_state = "infantry_legs"
-	soft_armor = list("melee" = 9, "bullet" = 12.5, "laser" = 12.5, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 9)
-	slowdown = 0.15
+	soft_armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 10)
```
Comment (Hughgent) [inline code/modules/modular_armor/armor_pieces.dm:136]: You are removing the variety for all leg options. is this desired?

File: code/modules/modular_armor/armor_pieces.dm [hunk 3]
```diff
@@ -171,22 +168,18 @@
...
 
 /obj/item/armor_module/armor/arms/marine
-	name = "\improper Jaeger Pattern Medium Infantry arm plates"
-	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides moderate protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece."
+	name = "\improper Jaeger Pattern Infantry arm plates"
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece."
 	icon_state = "infantry_arms"
-	soft_armor = list("melee" = 9, "bullet" = 12.5, "laser" = 12.5, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 9)
-	slowdown = 0.15
+	soft_armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 10)
```
Comment (Hughgent) [inline code/modules/modular_armor/armor_pieces.dm:174]: You are removing all variety from the arm options. Is this desired?

Comment (SplinterGP) [reply code/modules/modular_armor/armor_pieces.dm:174]: yeah, rengusta told to put it like that, same with legs

Comment (RenGusta) [reply code/modules/modular_armor/armor_pieces.dm:174]: Can confirm, helps with balance and keeping slowdown consistent and reasonable across armors

**Comment by Hughgent** [comment]
Please include in your change log the aspect that leg and arm armor pieces are now cosmetic.

**Comment by SplinterGP** [comment]
Done (y)

**Review by Hughgent — APPROVED** [review]

**Pull request merged**